### PR TITLE
add support for iPhone 5,3 9.3.1

### PR DESCRIPTION
**Pull request by retX0**
add offset for iPhone 5,3 9.3.1 and test pass on my own phone

### CODE REVIEW

**Comment by benjamin-42** [comment]
I merged it manually

File: .gitignore [hunk 1]
```diff
@@ -1,2 +1,3 @@
 .DS_Store
-xcuserdata
\ No newline at end of file
+xcuserdata
+.idea
```
Comment (benjamin-42) [inline .gitignore:3]: I prefer to only keep Xcode/macOS specific things, otherwise there will be a lot of things here

File: README.md [hunk 2]
```diff
@@ -101,3 +102,5 @@ Thanks: Lookout, Pangu team, i0n1c, jndok, kernelpool, planetbeing, qwertyoruiop
 > You feel it's true  
 > Its the little things in life  
 > that I feel
+
```
Comment (benjamin-42) [inline README.md:105]: I removed those extra blank lines

File: Trident.xcodeproj/project.pbxproj [hunk 3]
```diff
@@ -158,7 +158,7 @@
 				TargetAttributes = {
 					43CDFA421DCFA16600046EB0 = {
 						CreatedOnToolsVersion = 7.3.1;
-						DevelopmentTeam = 4P3PNUDM94;
+						DevelopmentTeam = 329FNU8Z9Y;
```
Comment (benjamin-42) [inline Trident.xcodeproj/project.pbxproj:161]: I didn't keep this change because it's specific to each person so it will be infinite changes for nothing

**Comment by retX0** [comment]
Thanks, This is my first to pull request, so there is some mistake, But ,,,, it`s wonderful experience.

**Pull request closed without merging**